docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
 
 ### Visualization
 
-- Support hardware anti-aliasing.
-- Thematic display of point clouds and terrain.
-- Support for compressed texture formats.
-- Large-scale scene animations (e.g., pedestrian simulations, weather animations, disaster simulations).
-- Support atmospheric rendering effects.
+- [X] Support hardware anti-aliasing.
+- [X] Thematic display of point clouds and terrain.
+- [ ] Support for compressed texture formats.
+- [X] Particle effects
+- [X] Screen-space effects
+- [ ] Large-scale scene animations (e.g., pedestrian simulations, weather animations, disaster simulations).
+- [ ] Support atmospheric rendering effects.
 
 ### Extensions
 
-- Enhance publishing workflow to Bentley's Extension Service.
-- Create examples and templates for building and publishing Extensions.
-- Support for better testing in the Extension's Hosting App, i.e. iTwin Viewer, iTwin Design Review, PlantSight, etc.
+- [ ] Enhance publishing workflow to Bentley's Extension Service.
```

Comment:
@calebmshafer @paulius-valiunas @SpencerWBarnes extensions?

Reply:
no one has touched the extension service for a while and the CLI tool was deleted too. Do we even have a publishing workflow at this point?

Reply:
@calebmshafer please update this to reflect our current Extension goals.